ui/components/app/snaps/snap-ui-address/snap-ui-address.tsx

```diff
@@ -41,15 +48,17 @@ export const SnapUIAddress: React.FunctionComponent<SnapUIAddressProps> = ({
     [caipIdentifier],
   );
 
-  const displayName = useDisplayName(parsed);
+  const name = useDisplayName(parsed);
 
-  const value =
-    displayName ??
-    shortenAddress(
-      parsed.chain.namespace === 'eip155'
-        ? toChecksumHexAddress(parsed.address)
-        : parsed.address,
-    );
+  // For EVM addresses, we make sure they are checksummed.
```

Comment:
Hmm how come we aren't checksumming for non-evm (at least the one that we know about -- bitcoin)?

Reply:
We are assuming that Snaps take care of this part and won't build in special checksum logic for additional chains